pinot-plugins/pinot-input-format/pinot-avro-base/src/main/java/org/apache/pinot/plugin/inputformat/avro/AvroUtils.java

```diff
@@ -332,6 +332,7 @@ private static void extractSchemaWithComplexTypeHandling(org.apache.avro.Schema
       default:
         DataType dataType = AvroSchemaUtil.valueOf(fieldType);
         addFieldToPinotSchema(pinotSchema, dataType, path, true, fieldTypeMap, timeUnit);
+        break;
```

Comment:
Is this break statement needed?

Reply:
this is required from the checkstyle rules